docs/logging/main-entities.md

```diff
@@ -0,0 +1,80 @@
+---
+id: tofu.logging.main.entities
+title: tofu.logging
+---
+
+
+# tofu.logging core concepts
+
+`tofu.logging` consists of three main things:
+
+## Typeclass `Loggable[A]`
+This is a type class that describes how an arbitrary instance of type A should be presented in the log messages.
+
+## Derivation
+Loggable instances could be derived automatically using `derevo`:
```

Comment:
I think it's better to put link to derevo's repo: https://github.com/tofu-tf/derevo